parquet-hadoop/src/test/java/org/apache/parquet/hadoop/util/MockHadoopInputStream.java

```diff
@@ -24,13 +24,13 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 
-class MockInputStream extends ByteArrayInputStream
+class MockHadoopInputStream extends ByteArrayInputStream
```

Comment:
This one implements Hadoop interfaces, `Seekable` and `PositionedReadable`, that are required for backing a `FSDataInputStream`. Those aren't available to `MockInputStream`. This could inherit from the other, but that would require moving classes around and exposing the test-jar, which doesn't seem worth it when we're only duplicating ~20 lines that may need to be changed independently in the future anyway.

Reply:
My bad, didn't realise the interfaces are in `parquet-hadoop`. I agree that exposing test-jar etc. would not worth it.